mosdef_cassandra/core/moves.py

```diff
@@ -173,6 +181,77 @@ def __init__(self, ensemble, species_topologies):
                 self.prob_translate += self.prob_rotate
                 self.prob_rotate = 0.0
 
+    def add_restricted_insertions(
+        self, species_topologies, restricted_type, restricted_value
+    ):
+        """Add restricted insertions for specific species and boxes
+
+        Parameters
+        ----------
+        species_topologies : list
+            list of ``parmed.Structures``, with one species per element
+        restricted_type : list
+            list of restricted insertion types, with one species per element.
+        restricted_value : list
+            list of restricted insertion values, with one species per element.
```

Comment:
I think these could be more clear; i.e. WRT the nesting of lists. One list per box and then one list per species.